examples/pie_and_polar_charts/polar_legend.py

```diff
@@ -9,21 +9,21 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-# radar green, solid grid lines
-plt.rc('grid', color='#316931', linewidth=1, linestyle='-')
-plt.rc('xtick', labelsize=15)
-plt.rc('ytick', labelsize=15)
-
-# force square figure and square axes looks better for polar, IMO
-fig = plt.figure(figsize=(8, 8))
-ax = fig.add_axes([0.1, 0.1, 0.8, 0.8],
-                  projection='polar', facecolor='#d5de9c')
+fig = plt.figure()
+ax = fig.add_subplot(projection='polar', facecolor='#d5de9c')
 
 r = np.arange(0, 3.0, 0.01)
```

Comment:
While we're at it, it would be nice to replace this with
```suggestion
r = np.linspace(0, 3, 301)
```
Makes the spirals go up to the 0°/180° line.

Reply:
done